garage/tf/policies/gaussian_mlp_policy_with_model2.py

```diff
@@ -0,0 +1,125 @@
+"""GaussianMLPPolicy with GaussianMLPModel."""
+import numpy as np
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+from garage.misc import logger
+from garage.tf.models.gaussian_mlp_model2 import GaussianMLPModel2
+from garage.tf.spaces import Box
+
+
+class GaussianMLPPolicyWithModel2:
+    """
+    GaussianMLPPolicy with GaussianMLPModel.
+
+    :param env_spec:
+    :param hidden_sizes: list of sizes for the fully-connected hidden
+    layers
+    :param learn_std: Is std trainable
+    :param init_std: Initial std
+    :param adaptive_std:
+    :param std_share_network:
+    :param std_hidden_sizes: list of sizes for the fully-connected layers
+     for std
+    :param min_std: whether to make sure that the std is at least some
+     threshold value, to avoid numerical issues
+    :param std_hidden_nonlinearity:
+    :param hidden_nonlinearity: nonlinearity used for each hidden layer
+    :param output_nonlinearity: nonlinearity for the output layer
+    :param mean_network: custom network for the output mean
+    :param std_network: custom network for the output log std
+    :param std_parametrization: how the std should be parametrized. There
+     are a few options:
+        - exp: the logarithm of the std will be stored, and applied a
+         exponential transformation
+        - softplus: the std will be computed as log(1+exp(x))
+    :return:
+    """
+
+    def __init__(self,
+                 env_spec,
+                 name='GaussianMLPPolicy',
+                 dist=tfp.distributions.MultivariateNormalDiag,
+                 *args,
```

Comment:
Could you specific args here, please? I want to find where [these params](https://github.com/rlworkgroup/garage/pull/512/files#diff-ee08069b2a03df62f0e02fcbbbf10d5cR97) are going to pass, but it's really hard to. 
Also this is the class user will be dealing direct with, this will make it very difficult to be sure what are the args and kwargs.

Reply:
Yes that will be more readable.